### ISSUE TITLE
can't reproduce the result

### ISSUE THREAD

**Issue by liangpingding** [youngbin-ro/Multi2OIE#3]
I run the project using the parameters you provided(some parameters you didn't provide, so I just use the default value in the code). The configuration of parameters are listed as follows:
{
    "seed": 1,
    "save_path": "./results",
    "bert_config": "bert-base-cased",
    "trn_data_path": "./datasets/openie4_train.pkl",
    "dev_data_path": [
        "./datasets/oie2016_dev.pkl",
        "./datasets/carb_dev.pkl"
    ],
    "dev_gold_path": [
        "./evaluate/OIE2016_dev.txt",
        "./carb/CaRB_dev.tsv"
    ],
    "max_len": 64,
    "device": "cuda",
    "visible_device": "2",
    "summary_step": 100,
    "use_lstm": false,
    "binary": false,
    "epochs": 1,
    "lstm_dropout": 0.0,
    "mh_dropout": 0.2,
    "pred_clf_dropout": 0.0,
    "arg_clf_dropout": 0.2,
    "batch_size": 64,
    "dev_batch_size": 128,
    "learning_rate": 3e-05,
    "n_arg_heads": 8,
    "n_arg_layers": 4,
    "pos_emb_dim": 64,
    "pred_n_labels": 3,
    "arg_n_labels": 9,
    "total_steps": 33791,
    "warmup_steps": 3379
}

And I got the best dev results in the first 1000 step and then the F1-score decreased. The results are shown in the following figure.
![image](https://user-images.githubusercontent.com/29528469/122329607-4f5b6100-cf64-11eb-916d-26280c39f99c.png)
 I'm wondering if there were some parameters I set wrong or if there are some mistakes for your provided parameters.

**Comment by youngbin-ro** [comment]
I apologize for the late reply.
Have you evaluated the model with the test data?
If the parameter is not specified, it is correct to use the default value.
Please try the test and let me know if the result is not as expected.

\+ I released the trained model, please check it.
- English Model: https://drive.google.com/file/d/11BaLuGjMVVB16WHcyaHWLgL6dg0_9xHQ/view?usp=sharing
- Multilingual Model: https://drive.google.com/file/d/1lHQeetbacFOqvyPQ3ZzVUGPgn-zwTRA_/view?usp=sharing
